src/test/java/com/commercetools/sync/commons/utils/ChannelCustomUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,54 @@
+package com.commercetools.sync.commons.utils;
+
+import com.commercetools.sync.channels.helpers.ChannelCustomActionBuilder;
+import com.commercetools.sync.inventories.InventorySyncOptionsBuilder;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.channels.commands.updateactions.SetCustomField;
+import io.sphere.sdk.channels.commands.updateactions.SetCustomType;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.commands.UpdateAction;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class ChannelCustomUpdateActionUtilsTest {
+
+    @Test
+    public void buildTypedSetCustomTypeUpdateAction_WithChannelResource_ShouldBuildChannelUpdateAction() {
+        final Channel channel = mock(Channel.class);
+        final Map<String, JsonNode> fieldsJsonMap = new HashMap<>();
+
+        final UpdateAction<Channel> updateAction =
+            GenericUpdateActionUtils.buildTypedSetCustomTypeUpdateAction("key",
+                fieldsJsonMap, channel, new ChannelCustomActionBuilder(), null, Channel::getId,
+                channelResource -> channelResource.toReference().getTypeId(),
+                channelResource -> null,
+                InventorySyncOptionsBuilder.of(mock(SphereClient.class)).build()).orElse(null);
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomType.class);
+    }
+
+    @Test
+    public void buildRemoveCustomTypeAction_WithChannelResource_ShouldBuildChannelUpdateAction() {
+        final UpdateAction<Channel> updateAction =
+            new ChannelCustomActionBuilder().buildRemoveCustomTypeAction(null, null);
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomType.class);
+    }
+
+    @Test
+    public void buildSetCustomFieldAction_WithChannelResource_ShouldBuildChannelUpdateAction() {
+        final UpdateAction<Channel> updateAction =
+            new ChannelCustomActionBuilder().buildSetCustomFieldAction(null, null, "name", mock(JsonNode.class));
+
+        assertThat(updateAction).isNotNull();
+        assertThat(updateAction).isInstanceOf(SetCustomField.class);
```

Comment:
check the type of the updateaction

Reply:
what do you mean?

Reply:
pls, don't duplicate you question/answer on the similar PR comments. The commenter makes you marks where you should take attention. Even if it is duplication of meaning, it has sense to show you all the places in the code where you should take attention.
But this doesn't mean the requestor has to duplicate the same question and flood other developers with useless notifications.